internal/testing/require/require.go

```diff
@@ -89,14 +101,41 @@ func CapturePanic(panics func()) (err error) {
 	return
 }
 
-func Same(t *testing.T, expected interface{}, actual interface{}, msgAndArgs ...interface{}) {
-	require.Same(t, expected, actual, msgAndArgs)
+func Same(t TestingT, expected interface{}, actual interface{}, formatWithArgs ...interface{}) {
+	require.Same(t, expected, actual, formatWithArgs)
 }
 
-func True(t *testing.T, value bool, msgAndArgs ...interface{}) {
-	require.True(t, value, msgAndArgs)
+// True fails if the actual value wasn't.
+//
+//  * formatWithArgs are optional. When the first is a string that contains '%', it is treated like fmt.Sprintf.
+func True(t TestingT, actual bool, formatWithArgs ...interface{}) {
+	if !actual {
+		fail(t, "expected true, but actual=false", formatWithArgs...)
+	}
 }
 
-func Zero(t *testing.T, i interface{}, msgAndArgs ...interface{}) {
-	require.Zero(t, i, msgAndArgs)
+func Zero(t TestingT, i interface{}, formatWithArgs ...interface{}) {
+	require.Zero(t, i, formatWithArgs)
+}
+
+// fail tries to treat the formatWithArgs as fmt.Sprintf parameters or joins on space.
+func fail(t TestingT, message string, formatWithArgs ...interface{}) {
```

Comment:
ps this is actually a bit smarter (IMHO) than testify as it detects a better message option